build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# runs source-deps and tests and then provided lein target with mranderson
+
+function check_result {
```

Comment:
looks like it. although what do you of of http://mywiki.wooledge.org/BashFAQ/105

btw @bbatsov surely will wlc a PR ;)